modules/ROOT/pages/java-ee-libraries.adoc

```diff
@@ -0,0 +1,27 @@
+= Java EE Libraries
+
+In the first Mule runtime version that exclusively supports Java 17 (Mule 4.9.0), connectors will no longer include Java EE libraries. Mule 4.6.0 through Mule 4.9.0 will continue to export Java EE libraries as part of the Container API only when Mule runtime runs on Java 8 or Java 11 to ensure backward compatibility. However, if Mule runtime runs on Java 17 or higher, you must include the Java EE libraries manually. For connectors that support multiple Java versions, such as Java 11 and Java 17, the Java EE libraries must match the versions included in Mule runtime. 
```

Comment:
Still not clear/correct:

* `if Mule runtime runs on Java 17 or higher, you must include the Java EE libraries manually.`: Connector developers don't know in which mule or java version the connector will be used on. Customers developing their connectors will rarely know it, Mulesoft and Partner developers will never do.
* `connectors will no longer include`-> connectors can no longer include

Reply:
* if Mule runtime runs on Java 17 or higher, you must include the Java EE libraries manually.

What do I replace this with?